lib/aca_entities/medicaid/ios/contracts/ssp_member__c_contract.rb

```diff
@@ -0,0 +1,700 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/ClassLength
+module AcaEntities
+  module Medicaid
+    module Ios
+      module Contracts
+        # Contract for SSP_Member__c
+        class SSPMemberCContract < Dry::Validation::Contract
+          params do
+            required(:FirstName__c).filled(:string)
+            required(:LATypeCode__c).filled(:string)
+            optional(:PhysicalAddressLine1__c).maybe(:string)
+            optional(:PhysicalCity__c).maybe(:string)
+            optional(:PhysicalStateCode__c).maybe(:string)
+            optional(:PhysicalZipCode5__c).maybe(:string)
+            required(:PreferredNotificationMethodCode__c).filled(:string)
+            required(:TaxFilerMemberCurrent__r).filled(:hash)
+            required(:LastName__c).filled(:string)
+            required(:MemberProgramsApplied__c).filled(:string)
+            required(:IsFederalRecognizedIndianTribeToggle__c).filled(:string)
+            required(:IsPregnant__c).filled(:string)
+            optional(:Name).maybe(:string)
```

Comment:
we have first name and last name, is this necessary ?

Reply:
Will throw this on the questions for the client list. assuming it is similar to enroll having a full name field for ease of reference